docs/howto/set-up-snapcraft.rst

```diff
@@ -0,0 +1,208 @@
+.. _install-snapcraft:
+
+Set up Snapcraft
+================
+
+Snapcraft can be installed on all popular Linux distributions with systemd,
+macOS, and Windows.
+
+Before you begin installation and setup, make sure your system meets the
+:ref:`system requirements <system-requirements>`.
+
+
+Install the main components
+---------------------------
+
+A working Snapcraft setup has two components -- Snapcraft itself, and a build
+provider.
+
+
+Install Snapcraft
+~~~~~~~~~~~~~~~~~
+
+.. tabs::
+
+  .. group-tab:: Linux
+
+    If your Linux distribution has snapd installed, the easiest way to install
+    Snapcraft is from the Snap Store:
+
+    .. code:: bash
+
+      sudo snap install snapcraft --classic
+
+    If your distribution lacks snapd or access to the Snap Store, but has
+    access to the Ubuntu or Debian package repository, you can install
+    Snapcraft as a deb package:
+
+    .. code:: bash
+
+      sudo apt install snapcraft
```

Comment:
This is only relevant on bionic and below and return outdated, unsupported versions. Our only official distribution on Linux right now is the snap.
```suggestion
```

Jon maintains an unofficial [nix package](https://search.nixos.org/packages?channel=24.11&show=snapcraft) though.